src/main/resources/org/jenkinsci/plugins/workflow/steps/PwdStep/help.html

```diff
@@ -1,3 +1,18 @@
 <div>
     Returns the current directory path as a string.
 </div>
+<br>
+<p>The <a href="https://www.jenkins.io/redirect/pipeline-snippet-generator">Pipeline Snippet Generator</a> generates this examples:
+
+  When the tmp argument is checked:
```

Comment:
I don't understand this one, is it to set tmp to true/false, or is it how to use pwd when tmp is true ? I think I don't fully get what is the 'tmp' value purpose on this step.

Reply:
Yeah, how to use pwd when tmp is true

Reply:
Would be good to add the default value here

Reply:
@kwhestone, I have added the default values.

Reply:
Since we now have `$WORKSPACE_TMP` predefined, this is unlikely to be used.